source/otus-business/src/main/java/br/org/otus/user/api/pendency/UserActivityPendencyFacade.java

```diff
@@ -0,0 +1,103 @@
+package br.org.otus.user.api.pendency;
+
+import br.org.otus.model.pendency.UserActivityPendency;
+import br.org.otus.response.exception.HttpResponseException;
+import br.org.otus.response.info.NotFound;
+import br.org.otus.service.pendency.UserActivityPendencyService;
+import org.ccem.otus.exceptions.webservice.common.DataNotFoundException;
+import org.ccem.otus.exceptions.webservice.common.MemoryExcededException;
+
+import javax.inject.Inject;
+import java.util.List;
+
+public class UserActivityPendencyFacade {
+
+  @Inject
+  private UserActivityPendencyService userActivityPendencyService;
+
+  public String create(String userEmail, String userActivityPendencyJson) {
+    return userActivityPendencyService.create(userEmail, userActivityPendencyJson).toString();
+  }
+
+  public void update(String userActivityPendencyId, String userActivityPendencyJson) {
+    try {
+      userActivityPendencyService.update(userActivityPendencyId, userActivityPendencyJson);
+    } catch (DataNotFoundException e) {
+      throw new HttpResponseException(NotFound.build(e.getMessage()));
+    }
+  }
+
+  public void delete(String userActivityPendencyId) {
+    try {
+      userActivityPendencyService.delete(userActivityPendencyId);
+    } catch (DataNotFoundException e) {
+      throw new HttpResponseException(NotFound.build(e.getMessage()));
+    }
+  }
+
+  public void deleteByActivityId(String activityId) {
+    try {
+      String pendencyId = getByActivityId(activityId).getId().toString();
+      userActivityPendencyService.delete(pendencyId);
+    } catch (DataNotFoundException e) {
+      throw new HttpResponseException(NotFound.build(e.getMessage()));
+    }
+  }
+
+  public UserActivityPendency getByActivityId(String activityId) {
+    try {
+      return userActivityPendencyService.getByActivityId(activityId);
+    } catch (DataNotFoundException e) {
+      throw new HttpResponseException(NotFound.build(e.getMessage()));
+    }
+  }
+
+  public List<UserActivityPendency> listAllPendenciesToReceiver(String receiverUserEmail) {
```

Comment:
@FlaviaAP diferentemente de linguagens não tipadas java é tipado, portanto a assinatura do metodo se faz utilizando todos os itens de sua declaração, modificador, retorno, nome e parametros. Neste caso não é preciso utilizar o list no nome, mas sim um verbo declarando seu objetivo. Ex. fetchPendenciesToReceiver()
Acredito que todas as assinaturas desta classe com esta caracteristica podem ser adaptadas.
Link clean code sobre nome assunto https://medium.com/coding-skills/clean-code-101-meaningful-names-and-functions-bf450456d90c